lib/hooks/useTime.ts

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+import { formatter } from '../utils';
+
+const useTime = () => {
+  const SunCalc = require('suncalc');
+  const [currentTime, setCurrentTime] = useState(formatter.format(Date.now()));
+  const [currentTimeNoFilter, setCurrentTimeNoFilter] = useState(Date.now());
+  const [isDaytime, setIsDaytime] = useState(false);
+  // get sunrise/sunset info for london
+  const times = SunCalc.getTimes(Date.now(), 51.5, -0.1);
```

Comment:
can we pass in lat, long to the useTime hook? then, wherever we use the hook, we can pass in the lat, long of the relevant site.